content/2.0.html.haml

```diff
@@ -167,6 +194,13 @@ title: "Jenkins 2.0"
   We're very interested in your feedback on what you think of the Jenkins 2.0
   preview releases.
 
+%p
+  To experience Jenkins 2.0 properly, we recommend you set up
+  %a{:href => 'https://wiki.jenkins-ci.org/display/JENKINS/Administering+Jenkins'}
+    a different Jenkins home directory
+  \. This way, you'll both get to try the new setup, and keep your production instance
```

Comment:
How can I get rid of the space between 'directory' and the period without making the period part of the link?